packages/playground/src/index.ts

```diff
@@ -14,15 +14,16 @@ type CustomObject = {
   object: object;
 };
 
+type AIInputObject = Record<string, object>;
+type InteractionFunction<Args extends {}> = (args: Args) => void;
```

Comment:
Most of the `InteractionFunction` doesn't return `void` right? It normally returns an object that's passed back to the server.